sdk/js/src/relayer/consts.ts

```diff
@@ -243,12 +267,16 @@ export const getWormscanAPI = (_network: Network) => {
 };
 
 export const getNameFromCCTPDomain = (
-  domain: number
+  domain: number,
+  environment: Network = "MAINNET"
```

Comment:
Because the new Sepolia chains have different chain ids, we need to check if we're in main or testnet environment before returning the name of the chain